code/components/citizen-server-impl/include/NoticeLogicProcessor.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <StdInc.h>
+#include <ServerInstanceBase.h>
+#include <ResourceManager.h>
+
+#include <json.hpp>
+
+using json = nlohmann::json;
+
+class NoticeLogicProcessor
```

Comment:
We'll be moving to namespacing everything, so best to apply it here as well.

Reply:
Would `fx` be the namespace of choice for this?

Reply:
yes that's fine, we'll get to others later when things are better designed